recipe/meta.yaml

```diff
@@ -113,12 +113,14 @@ outputs:
       host:
         - {{ pin_subpackage('_py-xgboost-mutex', exact=True) }}
         - {{ pin_subpackage('libxgboost', max_pin='x.x.x') }}
+        - libxgboost =*={{ string_prefix }}_h*_{{ PKG_BUILDNUM }}
```

Comment:
This includes the `string_prefix` and the `build/number`, which means we get the same generation of packages together